internal/types/metadata_test.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	"testing"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func TestMaintainer(t *testing.T) {
+	assert.Check(t, is.Equal(Maintainer{"foo", "foo@bar.com"}.String(), "foo <foo@bar.com>"))
+	assert.Check(t, is.Equal(Maintainer{"foo", ""}.String(), "foo <>"))
+	assert.Check(t, is.Equal(Maintainer{"foo", "bar"}.String(), "foo <bar>"))
+	assert.Check(t, is.Equal(Maintainer{"", ""}.String(), " <>"))
+}
+
+func TestMaintainers(t *testing.T) {
+	m1 := Maintainer{"foo", "foo@bar.com"}
+	m2 := Maintainer{"bar", "bar@baz.com"}
+	for _, tc := range []struct {
+		maintainers Maintainers
+		expected    string
+	}{
+		{
+			maintainers: []Maintainer{},
+			expected:    "",
+		},
+		{
+			maintainers: []Maintainer{m1},
+			expected:    "foo <foo@bar.com>",
+		},
+		{
+			maintainers: []Maintainer{m1, m2},
+			expected:    "foo <foo@bar.com>,bar <bar@baz.com>",
```

Comment:
It feels we're missing a space after the comma, no?

Reply:
yes in code there is :angel: I can do the change :wink: